ds3_java_cli/src/test/java/com/spectralogic/ds3cli/Ds3Cli_Test.java

```diff
@@ -638,9 +640,8 @@ public void putObjectWithSyncNotSupportedVersion() throws Exception {
         PowerMockito.mockStatic(BlackPearlUtils.class);
 
         final Ds3Cli cli = new Ds3Cli(new Ds3ProviderImpl(client, helpers), args, mockedFileUtils);
-        final CommandResponse result = cli.call();
-        assertThat(result.getMessage(), is("Failed: The sync command is not supported with your version of BlackPearl."));
-        assertThat(result.getReturnCode(), is(0));
+        cli.call(); //should throw SyncNotSupportedException
+        fail();
```

Comment:
The fail is not required.  The test will fail since `expected` is being used above.